meta/types/storage-class.go

```diff
@@ -11,23 +11,23 @@ const (
 	// ObjectStorageClassStandard is a ObjectStorageClass enum value
 	ObjectStorageClassStandard StorageClass = iota
 
-	// ObjectStorageClassReducedRedundancy is a ObjectStorageClass enum value
-	ObjectStorageClassReducedRedundancy
```

Comment:
Not need to change

Reply:
I use enumeration order when resolving conflicts；e.x：GLACIER > STANDARD_IA > STANDARD